From bbd478b6a9abc7fe6a08a7a3201fb0e97326b215 Mon Sep 17 00:00:00 2001
From: Dmitry Ornatsky <cdo021c@motorola.com>
Date: Tue, 18 Aug 2015 10:56:52 -0500
Subject: [PATCH 025/959] IKKRNBSP-3332 arm/dt: 8952: configure lcd

Set max WLED current to 20 mA. Enable display regulators
in PMI8952.

Change-Id: I316382bb20d77f7d426d9223f74650688a4d15bd
Signed-off-by: Dmitry Ornatsky <cdo021c@motorola.com>
Reviewed-on: http://gerrit.mot.com/777389
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index 2f07f426b89..e0f79fed587 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -97,3 +97,16 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart_console_sleep>;
 };
+
+&spmi_bus {
+	qcom,pmi8950@3 {
+		/* display boost and inverting boost regulators */
+		labibb: qpnp-labibb-regulator {
+			status = "ok";
+		};
+		qcom,leds@d800 {
+			qcom,fs-curr-ua = <20000>;
+			/delete-property/ qcom,en-ext-pfet-sc-pro;
+		};
+	};
+};
-- 
2.11.0

